lib/src/pipes/random_colorize_pipe.dart

```diff
@@ -0,0 +1,31 @@
+import "package:angular2/di.dart" show PipeTransform, Pipe;
+import 'package:angular2/src/common/pipes/invalid_pipe_argument_exception.dart';
+
+/// Based on [seed] provided to the pipe, return a css consumable color
+///
+/// Returned format: `#somecolor`
+///
+/// __Usage__:
+///
+///     <div [style.background-color]="1234 | randomColorize"></div>
+///
+/// __Limitations__:
+///
+/// Can only accept seeds of `String` and `int` types
+@Pipe('randomColorize')
+class SkawaRandomColorizePipe implements PipeTransform {
+  String transform(dynamic seed) {
+    if (!_supportedInput(seed)) {
+      throw new InvalidPipeArgumentException(SkawaRandomColorizePipe, seed);
+    }
+    num hash;
+    if (seed is String) {
+      hash = seed.hashCode % 1000000;
+    } else if (seed is num) {
+      hash = seed % 1000000;
```

Comment:
This won't return a valid css color if there are less than 6 digits in the seed.

Reply:
due to the  mdn [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/color) it is still a valid color if its 3,4 6 or 8 digits number and changed to code to correspondig to that